src/test/java/de/unistuttgart/iste/gits/skilllevel_service/SkillLevelServiceTest.java

```diff
@@ -277,7 +278,14 @@ void testRepetitionNotEnoughTimePassed() {
                         .build()
         );
 
-        when(contentServiceClient.getContentsWithUserProgressData(any(), any())).thenReturn(contents);
+        when(contentServiceClient.getAssessmentsOfChapter(any(), any())).thenReturn(
+                contents.stream()
+                        .map(x -> new ContentServiceClient.GenericAssessmentResponse(
```

Comment:
if you add a constructor to "GenericAssessmentResponse" that accepts a "Assessment" you don't need this construct.

Reply:
also not necessary if the ContentResponse class implements Content